test/ssl-tests/14-groups.conf.in

```diff
@@ -10,7 +10,7 @@ use warnings;
 use OpenSSL::Test;
 use OpenSSL::Test::Utils qw(anydisabled);
 
-my @curves = ("sect163k1", "sect163r1", "sect163r2", "sect193r1",
+my @groups = ("sect163k1", "sect163r1", "sect163r2", "sect193r1",
```

Comment:
I also think this renaming is wrong.  It's a CURVE, even though the API handles curves and groups.